google/cloud/forseti/services/inventory/base/cai_gcp_client.py

```diff
@@ -1148,6 +1148,110 @@ def iter_crm_projects(self, parent_type, parent_id):
         for project in resources:
             yield project
 
+    def iter_crm_organization_access_policies(self, org_id):
+        """Iterate access policies in an organization from Cloud Asset data.
+
+        Args:
+            org_id (str): id of the organization to get the Policy.
+
+        Yields:
+            dict: Generator of access policies for an organization.
+        """
+        resources = self.dao.iter_cai_assets(
+            ContentTypes.access_policy,
+            'cloudresourcemanager.googleapis.com/Organization',
+            '//cloudresourcemanager.googleapis.com/{}'.format(org_id),
+            self.engine)
+        for org in resources:
```

Comment:
do you mean access_policy instead of org?

Reply:
Yeah updated